server/dredd.yml

```diff
@@ -2,6 +2,7 @@ dry-run: null
 hookfiles: ./**/*.hook.js
 language: nodejs
 sandbox: false
+server: yarn dev:test
```

Comment:
@AleksSenkou kind of tricky 🙂 What do you think about using `test` without `dev` -> `yarn test`?

Reply:
@DDKatch that's kind of rails/sequalize/... pattern of tasks `domain:task` style 😉 

`yarn test` is used for running tests everywhere, it would be not clear

Reply:
@DDKatch play with naming for sure! let us know if something else comes to your mind

Reply:
@AleksSenkou got it